src/ingest_validation_tools/directory-schemas/imc3d.yaml

```diff
@@ -1,13 +1,13 @@
 # mcd/:
 -
-  pattern: 'mcd/[^/]+_HuBMAP_[^/]+organ[^/]+_slide[^/]+\.zip'
-  description: 'csv containing labels for sections as well as whether or not they were included in the 3D model'
+  pattern: 'mcd/[^/]+_HuBMAP_[^/]+_slide[^/]+\.zip'
+  description: 'CSV containing labels for sections as well as whether or not they were included in the 3D model'
```

Comment:
If the zipped file is a csv, can we say the the extension should really be: `...csv.zip`? It's conventional to indicate what a `.zip` or `.gz` is once you unzip it.

Reply:
The panel.csv should be preserved in the "extras" folder because:

- the panel.csv is provided by Zurich and is valued as an "original" source of information.
- the tag-antibody-targets in the panel.csv are ordered by the atomic weight of each metal tag which represents the order of the peaks in the mass spec data. Even though the ID info is somewhat redundant info provided in the antibodies TSV and the channelnames_report, it is considered important by the data providers.

Reply:
@cebriggs7135 :
- If this would be useful for future users, put it in the PR. Information that's just in the discussion here will be hard for them to find.

@icaoberg : 
- Chris says this should be be in extras -- please update the PR if that is correct. 
- At a minimum, what are the columns the the CSV?
- If we're still in a phase where we can nudge the TMC, please ask for the file to end `.csv.zip`... but if it's too late for that, please say so in the PR and I won't nag.

Reply:
@mccalluc 

1. That is correct. `panels.csv` is now in `extras`.
2. Descriptions have been updated.
3. Too late, but we should add this recommendation to the guide. However, in terms of the reg expression, can we do `.zip|.csv.zip`? Would that be an acceptable practice?